src/main/resources/jupyter/edit-mode.js

```diff
@@ -401,7 +401,5 @@ define(() => {
         location.reload(true)
     }
 
-    return {
-        load_ipython_extension: init
-    }
+    init()
```

Comment:
Interesting  - can anyone explain how these files are executed? I tried to trace this in `init-actions.sh` but couldn't figure it out. These don't seem to have the normal Jupyter JS extension setup.

Reply:
```extension_entry.js``` imports the extensions. How it does so, I'm not sure. In ```init-actions.sh``` it indicates the fact it is named custom.js is important, which tells me something downstream imports it, but I haven't had any luck finding what beyond that, I just followed how google_sign_in.js is handled.

Reply:
Yeah https://github.com/DataBiosphere/leonardo/blob/develop/src/main/resources/jupyter/extension_entry.js is the entrypoint to these internal extensions. It is injected into Jupyter via the `custom.js` hook.

Note these are different than user-supplied extensions, which are installed by `jupyter nbextension install ...`.